wiki/Gameplay/Hit_object/Slider/es.md

```diff
@@ -14,17 +14,17 @@ tags:
 
 # Slider
 
-Un *slider* es un [objeto](/wiki/Gameplay/Hit_object) en [osu!](/wiki/Game_mode/osu!), que consta de un sliderhead, un sliderbody y un slidertail. Una vez que el círculo de aproximación alcanza el borde del sliderhead, como con los [círclulos](/wiki/Gameplay/Hit_object/Hit_circle), el jugador debe hacer clic/tocar el comienzo del slider y luego, manteniendo presionado el botón, seguir una bola en movimiento (llamada bola del slider) a lo largo de la pista hasta que se alcance el slidertail. Si hay una flecha inversa, el jugador debe seguir la bola del slider hacia atrás por el mismo camino. Los sliders con flechas invertidas se denominan [sliders inversos](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). La velocidad de un slider se define por su [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) y el multiplicador de velocidad del slider de la sección de tiempo actual.
+Un *slider* es un [objeto](/wiki/Gameplay/Hit_object) en [osu!](/wiki/Game_mode/osu!), que consta de un sliderhead, un sliderbody y un slidertail. Una vez que el círculo de aproximación alcanza el borde del sliderhead, como con los [círclulos](/wiki/Gameplay/Hit_object/Hit_circle), el jugador debe hacer clic/tocar el comienzo del slider y luego, mantener presionado el botón, seguir una bola en movimiento (llamada bola del slider) a lo largo de la pista hasta que se alcance el slidertail. Si hay una flecha inversa, el jugador debe seguir la bola del slider hacia atrás por el mismo camino. Los sliders con flechas invertidas se denominan [sliders inversos](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). La velocidad de un slider se define por su [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) y el multiplicador de velocidad del slider de la sección de tiempo actual.
```

Comment:
```suggestion
Un *slider* es un [objeto](/wiki/Gameplay/Hit_object) en [osu!](/wiki/Game_mode/osu!), que consta de un sliderhead, un sliderbody y un slidertail. Una vez que el círculo de aproximación alcanza el borde del sliderhead, como con los [círclulos](/wiki/Gameplay/Hit_object/Hit_circle), el jugador debe hacer clic/tocar el comienzo del slider y luego, manteniendo presionado el botón, seguir una bola en movimiento (llamada bola del slider) a lo largo de la pista hasta que se alcance el slidertail. Si hay una flecha inversa, el jugador debe seguir la bola del slider hacia atrás por el mismo camino. Los sliders con flechas invertidas se denominan [sliders inversos](/wiki/Gameplay/Hit_object/Slider/Reverse_slider). La velocidad de un slider se define por su [velocidad del slider](/wiki/Gameplay/Hit_object/Slider/Slider_velocity) y el multiplicador de velocidad del slider de la sección de tiempo actual.
```

It's important keeping the gerund as you're following the ball *while* you're holding the button. Not just pressing it and releasing it.